--- a/Code/GameEngine/Include/Common/NameKeyGenerator.h
+++ b/Code/GameEngine/Include/Common/NameKeyGenerator.h
@@ -34,6 +34,7 @@
 #define __NAMEKEYGENERATOR_H_
 
 #include "Lib/BaseType.h"
+#include "Common/NameKeyType.h"
 #include "Common/SubsystemInterface.h"
 #include "Common/GameMemory.h"
 #include "Common/AsciiString.h"
@@ -46,12 +47,7 @@
 	determined at runtime. (The generated code is basically identical, of course.)
 */
 //------------------------------------------------------------------------------------------------- 
-enum NameKeyType
-{
-	NAMEKEY_INVALID					= 0,
-	NAMEKEY_MAX							= 1<<23,					// max ordinal value of a NameKey (some code relies on these fitting into 24 bits safely)
-	FORCE_NAMEKEYTYPE_LONG	= 0x7fffffff	// a trick to ensure the NameKeyType is a 32-bit int
-};
+
 
 //-------------------------------------------------------------------------------------------------
 /** A bucket entry for the name key generator */
