--- a/Code/GameEngine/Include/GameClient/View.h
+++ b/Code/GameEngine/Include/GameClient/View.h
@@ -28,6 +28,7 @@
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
 #pragma once
+#include "Common/ViewFilterDefinitions.h"
 
 #ifndef _VIEW_H_
 #define _VIEW_H_
@@ -36,7 +37,7 @@
 #include "Common/GameType.h"
 #include "Common/Snapshot.h"
 #include "Lib/BaseType.h"
-#include "WW3D2/ColType.h"			///< we don't generally do this, but we need the W3D collision types
+#include "WW3D2/coltype.h"			///< we don't generally do this, but we need the W3D collision types
 
 #define DEFAULT_VIEW_WIDTH 640
 #define DEFAULT_VIEW_HEIGHT 480
