--- a/wwlib/always.h
+++ b/wwlib/always.h
@@ -41,6 +41,11 @@
 #define ALWAYS_H
 
 #include <assert.h>
+#include <stddef.h>
+#include <new>
+#ifndef __cdecl
+#define __cdecl
+#endif
 
 // Disable warning about exception handling not being enabled. It's used as part of STL - in a part of STL we don't use.
 #pragma warning(disable : 4530)
@@ -77,12 +82,14 @@
 
 	#define _OPERATOR_NEW_DEFINED_
 
+#if !defined(__GNUC__)
 	extern void * __cdecl operator new		(size_t size);
 	extern void __cdecl operator delete		(void *p);
 
 	extern void * __cdecl operator new[]	(size_t size);
 	extern void __cdecl operator delete[]	(void *p);
 
+#endif
 	// additional overloads to account for VC/MFC funky versions
 	extern void* __cdecl operator new			(size_t nSize, const char *, int);
 	extern void __cdecl operator delete		(void *, const char *, int);
@@ -93,8 +100,11 @@
 	// additional overloads for 'placement new'
 	//inline void* __cdecl operator new							(size_t s, void *p) { return p; }
 	//inline void __cdecl operator delete						(void *, void *p)		{ }
+#if !defined(__GNUC__)
 	inline void* __cdecl operator new[]						(size_t s, void *p) { return p; }
 	inline void __cdecl operator delete[]					(void *, void *p)		{ }
+
+#endif
 
 #endif
 
@@ -235,7 +245,7 @@
 #endif
 
 #if defined(__WATCOMC__)
-#include	"watcom.h"
+#include "WATCOM.H"
 #endif
 
 
