--- a/Code/Libraries/Source/WWVegas/WW3D2/vertmaterial.h
+++ b/Code/Libraries/Source/WWVegas/WW3D2/vertmaterial.h
@@ -271,7 +271,7 @@
 	** Apply the render states corresponding to a NULL vetex material to D3D
 	*/
 	static void			Apply_Null(void);
-	unsigned long		VertexMaterialClass::Compute_CRC(void) const;
+	unsigned long		Compute_CRC(void) const;
 
 	static VertexMaterialClass *Presets[PRESET_COUNT];
 };
