--- a/Code/Libraries/Source/WWVegas/WWMath/lookuptable.cpp
+++ b/Code/Libraries/Source/WWVegas/WWMath/lookuptable.cpp
@@ -38,7 +38,7 @@
 
 #include "lookuptable.h"
 #include "curve.h"
-#include "wwfile.h"
+#include "WWFILE.H"
 #include "ffactory.h"
 #include "chunkio.h"
 #include "persistfactory.h"
@@ -145,7 +145,7 @@
 	// check if we already have this table loaded...
 	RefMultiListIterator<LookupTableClass> it(&Tables);
 	for (it.First(); !it.Is_Done(); it.Next()) {
-		if (stricmp(it.Peek_Obj()->Get_Name(),name) == 0) {
+		if (strcasecmp(it.Peek_Obj()->Get_Name(),name) == 0) {
 			return it.Get_Obj(); // add a reference for the user...
 		}
 	}
