--- a/Code/Libraries/Source/WWVegas/WWLib/bittype.h
+++ b/Code/Libraries/Source/WWVegas/WWLib/bittype.h
@@ -41,26 +41,28 @@
 #ifndef BITTYPE_H
 #define BITTYPE_H
 
+#include <stdint.h>
+
 typedef unsigned char	uint8;
 typedef unsigned short	uint16;
-typedef unsigned long	uint32;
+typedef uint32_t	uint32;
 typedef unsigned int    uint;
 
 typedef signed char		sint8;
 typedef signed short		sint16;
-typedef signed long		sint32;
+typedef int32_t		sint32;
 typedef signed int      sint;
 
 typedef float				float32;
 typedef double				float64;
 
-typedef unsigned long   DWORD;
+typedef uint32_t        DWORD;
 typedef unsigned short	WORD;
 typedef unsigned char   BYTE;
 typedef int             BOOL;
 typedef unsigned short	USHORT;
 typedef const char *		LPCSTR;
 typedef unsigned int    UINT;
-typedef unsigned long   ULONG;
+typedef uint32_t        ULONG;
 
 #endif //BITTYPE_H
