--- a/Code/Libraries/Source/WWVegas/WWMath/curve.h
+++ b/Code/Libraries/Source/WWVegas/WWMath/curve.h
@@ -43,11 +43,11 @@
 #define CURVE_H
 
 #ifndef ALWAYS_H
-#include "always.h"
+#include "../WWLib/always.h"
 #endif
 
 #ifndef VECTOR_H
-#include "vector.h"
+#include "../WWLib/Vector.H"
 #endif
 
 #ifndef VECTOR3_H
@@ -55,7 +55,7 @@
 #endif
 
 #ifndef PERSIST_H
-#include "persist.h"
+#include "../WWSaveLoad/persist.h"
 #endif
 
 class ChunkLoadClass;
