--- a/wwlib/realcrc.cpp
+++ b/wwlib/realcrc.cpp
@@ -40,8 +40,10 @@
 #include "realcrc.h"
 #include <ctype.h>
 
+static_assert(sizeof(uint32_t) == 4, "Original checksum width");
+
 //    CRC for poly 0x04C11DB7   
-unsigned long  CRC32_Table[ 256 ] =
+uint32_t  CRC32_Table[ 256 ] =
 {
 	0x00000000L, 0x77073096L, 0xEE0E612CL, 0x990951BAL, 
 	0x076DC419L, 0x706AF48FL, 0xE963A535L, 0x9E6495A3L,
@@ -109,7 +111,7 @@
 	0xB40BBE37L, 0xC30C8EA1L, 0x5A05DF1BL, 0x2D02EF8DL
 };
 
-#define CRC32(c,crc) (CRC32_Table[((unsigned long)(crc) ^ (c)) & 0xFFL] ^ (((crc) >> 8) & 0x00FFFFFFL))
+#define CRC32(c,crc) (CRC32_Table[((uint32_t)(crc) ^ (c)) & 0xFFL] ^ (((crc) >> 8) & 0x00FFFFFFL))
 
 
 /***********************************************************************************************
@@ -123,7 +125,7 @@
  *                                                                                             *
  * HISTORY:                                                                                    *
  *=============================================================================================*/
-unsigned long	CRC_Memory( const unsigned char *data, unsigned long length, unsigned long crc )
+uint32_t	CRC_Memory( const unsigned char *data, uint32_t length, uint32_t crc )
 {
  	crc ^= 0xFFFFFFFF;									// invert previous CRC
 	while ( length-- ) {
@@ -144,7 +146,7 @@
  *                                                                                             *
  * HISTORY:                                                                                    *
  *=============================================================================================*/
-unsigned long	CRC_String( const char *string, unsigned long crc )
+uint32_t	CRC_String( const char *string, uint32_t crc )
 {
  	crc ^= 0xFFFFFFFF;									// invert previous CRC
 	while ( *string )	{
@@ -165,11 +167,11 @@
  *                                                                                             *
  * HISTORY:                                                                                    *
  *=============================================================================================*/
-unsigned long	CRC_Stringi( const char *string, unsigned long crc )
+uint32_t	CRC_Stringi( const char *string, uint32_t crc )
 {
  	crc ^= 0xFFFFFFFF;									// invert previous CRC
 	while ( *string )	{
-		char c = (char)toupper(*string++);
+		char c = (char)toupper((unsigned char)*string++);
 		crc = CRC32( c, crc );				// calc crc for each byte
 	}
 	return (crc ^ 0xFFFFFFFF); 						// invert new CRC and return it
