--- a/Code/Libraries/Source/WWVegas/WW3D2/dx8renderer.h
+++ b/Code/Libraries/Source/WWVegas/WW3D2/dx8renderer.h
@@ -49,7 +49,7 @@
 #include "wwstring.h"
 #include "simplevec.h"
 #include "refcount.h"
-#include "vector.h"
+#include "Vector.H"
 #include "dx8list.h"
 #include "shader.h"
 #include "dx8wrapper.h"
@@ -158,7 +158,7 @@
 	bool														AnyDelayedPassesToRender;
 
 	void Generate_Texture_Categories(Vertex_Split_Table& split_table,unsigned vertex_offset);
-	void DX8FVFCategoryContainer::Insert_To_Texture_Category(
+	void Insert_To_Texture_Category(
 		Vertex_Split_Table& split_table,
 		TextureClass** textures,
 		VertexMaterialClass* mat,
