--- a/Code/Libraries/Source/WWVegas/WWLib/ffactory.cpp
+++ b/Code/Libraries/Source/WWVegas/WWLib/ffactory.cpp
@@ -34,7 +34,7 @@
  * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
 #include	"ffactory.h"
-#include	"rawfile.h"
+#include "RAWFILE.H"
 #include "bufffile.h"
 #include "realcrc.h"
 #include	<stdio.h>
@@ -293,7 +293,7 @@
 					}
 				}
 			} else {
-				new_name.Format("%s%s",SubDirectory,stripped_name);
+				new_name.Format("%s%s",SubDirectory.Peek_Buffer(),stripped_name.Peek_Buffer());
 			}
 		}
 
